src/puppetlabs/ezbake/core.clj

```diff
@@ -286,6 +287,17 @@ Dependency tree:
         (deputils/cp-files-of-type lein-project "doc"
                                    docs-prefix get-out-dir-for-doc-file)))
 
+(defn deploy-snapshot
+  "Given a project map with a snapshot version, deploy to the configured
+  snapshots repository and return the version of the deployed artifact. If given
+  a map with a non-snapshot version, does nothing and returns nil."
+  [lein-project]
+  (when (deputils/snapshot-version? (:version lein-project))
+    (-> (deploy/deploy lein-project)
+      .getArtifacts
+      (nth 0)
```

Comment:
Yes. `.deploy` returns a [DeployResult](https://sonatype.github.io/sonatype-aether/apidocs/index.html?org/sonatype/aether/util/version/GenericVersionScheme.html) which only contains the artifacts that were deployed.